.github/workflows/format.yml

```diff
@@ -42,36 +42,27 @@
 # LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 # THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #  DO-NOT-REMOVE end-copyright-block
+name: Lint with clang-format and push any changes
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Should we also add `dev` to the branch list?

Reply:
Yes, I'll fix this

Reply:
This file no longer exists because I moved the `lint` job as a dependency for all other build jobs.

All jobs now share the same `master` and `dev` branch hooks.